### PR TITLE
let's try upcasting rmsnorm

### PR DESCRIPTION
**Pull request by dlwh**

### CODE REVIEW

File: src/levanter/models/llama.py [hunk 1]
```diff
@@ -305,17 +305,20 @@ def update_state_dict(self, state_dict: StateDict, prefix: Optional[str] = None)
...
+    axis: AxisSpec = eqx.static_field()
+    weight: Optional[NamedArray]
+    bias: Optional[NamedArray]
+
+    eps: float = eqx.static_field(default=1e-5)
+    dtype: Optional[jnp.dtype] = eqx.static_field(default=jnp.float32)
 
     @staticmethod
-    def init(axis: AxisSpec, eps: float = 1e-6, use_weight: bool = True, use_bias: bool = False):
+    def init(axis: AxisSpec, eps: float = 1e-5, use_weight: bool = True, use_bias: bool = True, dtype=jnp.float32):
```
Comment (Ivan-Zhou) [inline src/levanter/models/llama.py:321]: What's your rationale behind these changes to the default value?

Llama's implementation doesn't have bias, right?
https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L88

The default rms eps is 1e-6: 
https://huggingface.co/docs/transformers/en/model_doc/llama2#transformers.LlamaConfig

**Review by Ivan-Zhou — COMMENTED** [review]
Do you want me to try this change on our Llama 1B run?

File: src/levanter/models/llama.py (hunk 1 above)
Comment (dlwh) [reply src/levanter/models/llama.py:321]: copypasta

Comment (dlwh) [inline src/levanter/models/llama.py:321]:
```suggestion
    def init(axis: AxisSpec, eps: float = 1e-6, use_weight: bool = True, use_bias: bool = True, dtype=jnp.float32):
```

**Comment by dlwh** [comment]
@Ivan-Zhou yes please!

**Pull request merged**